conf/cms/settings_local.py

```diff
@@ -0,0 +1,9 @@
+
+ALLOWED_HOSTS = ['cep.test']
+CEP_AUTH_VERIFICATION_ENDPOINT = 'http://core:6000'
```

Comment:
fyi. @taoteg .  

two changes that fixed the login error that we had previously and gave up on:
* here we got to have the endpoint as the name of the service in the docker-compose
* and then the INCLUDES_CORE_PORTAL is set to False in settings_local.py included in the CMS image.  So we have to ensure we overwrite that file so that that variable stays as True.  So see the change in the docker-compose.yml